src/deployment/deployment-hapi-plugin.ts

```diff
@@ -146,20 +144,19 @@ class DeploymentHapiPlugin {
     }, {
       method: 'GET',
       path: '/raw-deployment-handler/{param*}',
-      handler: this.directoryHandler('key'),
+      handler: this.directoryHandler(PREKEY),
       config: {
         auth: 'customAuthorize',
```

Comment:
I'm going through the whole file to better understand how this plugin works, and have a few comments that are not related to this PR per se, but wanted to leave them in any case:

- The `auth` boolean that's passed to this function could maybe be a bit more descriptive. What does it exactly mean in this function's context?
- No need to necessarily react to this, but mapping over the array at the end and overriding the `config.auth` option seems a bit error-prone. You might set a value for `config.auth` (or leave it empty, i.e. enabling auth) and not realize that it's actually disabled depending on the parameter. It also took a little bit of thinking to understand what was being done with the map, but this is related to the first point.